### PR TITLE
Adding the requirements in feature-a 

### PR DESCRIPTION
**Pull request by Aliahmad-karimi**
Add the file .gitignore with adding the test.md in it
 Add the linters in the folder
 Add the h1 tag with hello microverse in index.html
 Add the color to h1 in style.css

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (topeogunleye) [inline .gitignore]: - [ ] Great job adding test.md to your `.gitignore` file, however I kindly suggest that you add `node_modules/` also to your .gitignore file as written in the project requirements.

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```
Comment (topeogunleye) [inline .github/workflows/linters.yml:2]: - [ ] Great job setting up your linters correctly, however pls kindly ensure that all your linter results are green i.e Linter should report NO errors. you can make run `npx stylelint --fix "**/*.{css,scss}"` to fix stylelint errors if you have too many of them, but try to fix them line by line also to learn best practices.
image

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Aliahmad-karimi ,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
image
> ### Good Commit Messages:
>- [x] Add the file .gitignore with adding the test.md in it
>- [x] Add the linters in the folder
>- [x] Add the h1 tag with hello microverse in index.html
>- [x] Add the color to h1 in style.css
>- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);
>
> ## DESCRIPTIVE TITLE AND DESCRIPTION
>- [ ] Please kindly add a more descriptive PR title and description.
> See an example below:
>Screenshot from 2022-04-19 17-33-33

> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Aliahmad-karimi** [comment]
Dear Reviewer
I fixed 2 errors on [stylelint] but one of them is a little confusing I did not get that one. please help me out. 
I did the rest of your comments and appreciate all of the points.

**Comment by Aliahmad-karimi** [comment]
Dear 
I have fixed all of the errors.

**Review by omarsalem7 — APPROVED** [review]
Hi @Aliahmad-karimi 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
> Dear Reviewer I fixed 2 errors on [stylelint] but one of them is a little confusing I did not get that one. please help me out. I did the rest of your comments and appreciate all of the points.

Great job fixing the linter errors. :+1:

**Pull request merged**